paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/BancontactSpec.kt

```diff
@@ -53,7 +52,7 @@ internal val bancontactEmailSection =
 internal val bancontactMandate = StaticTextSpec(
     IdentifierSpec.Generic("mandate"),
     R.string.stripe_paymentsheet_sepa_mandate,
-    Color.Gray
```

Comment:
Now that we have this, I wonder if we should use Color.Gray/Color.Black/Color.lightGray at all

Reply:
I think our preferred way of going forward should be passing the ResID rather than compose colors. Just so changes in color don't come with big 10 file changes like this one ;)